<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="4" total="4" passed="0">
  <reporter-output>
    <line>
      <![CDATA[App Link:-https://www.surveymonkey.com/]]>
    </line>
    <line>
      <![CDATA[Before Suite Execution Finish]]>
    </line>
    <line>
      <![CDATA[Unable to launch browser instance due to following exception : Could not start a new session. Possible causes are invalid address of the remote server or browser start-up failure.
Build info: version: '2.48.2', revision: '41bccdd10cf2c0560f637404c2d96164b67d9d67', time: '2015-10-09 13:08:06'
System info: host: 'D1ML14605', ip: '192.168.1.3', os.name: 'Windows 8', os.arch: 'amd64', os.version: '6.2', java.version: '1.7.0_45'
Driver info: driver.version: RemoteWebDriver]]>
    </line>
    <line>
      <![CDATA[Unable to launch browser instance due to following exception : Could not start a new session. Possible causes are invalid address of the remote server or browser start-up failure.
Build info: version: '2.48.2', revision: '41bccdd10cf2c0560f637404c2d96164b67d9d67', time: '2015-10-09 13:08:06'
System info: host: 'D1ML14605', ip: '192.168.1.3', os.name: 'Windows 8', os.arch: 'amd64', os.version: '6.2', java.version: '1.7.0_45'
Driver info: driver.version: RemoteWebDriver]]>
    </line>
    <line>
      <![CDATA[Unable to launch browser instance due to following exception : Could not start a new session. Possible causes are invalid address of the remote server or browser start-up failure.
Build info: version: '2.48.2', revision: '41bccdd10cf2c0560f637404c2d96164b67d9d67', time: '2015-10-09 13:08:06'
System info: host: 'D1ML14605', ip: '192.168.1.3', os.name: 'Windows 8', os.arch: 'amd64', os.version: '6.2', java.version: '1.7.0_45'
Driver info: driver.version: RemoteWebDriver]]>
    </line>
    <line>
      <![CDATA[Unable to launch browser instance due to following exception : Could not start a new session. Possible causes are invalid address of the remote server or browser start-up failure.
Build info: version: '2.48.2', revision: '41bccdd10cf2c0560f637404c2d96164b67d9d67', time: '2015-10-09 13:08:06'
System info: host: 'D1ML14605', ip: '192.168.1.3', os.name: 'Windows 8', os.arch: 'amd64', os.version: '6.2', java.version: '1.7.0_45'
Driver info: driver.version: RemoteWebDriver]]>
    </line>
    <line>
      <![CDATA[After Suite Execution Finish]]>
    </line>
  </reporter-output>
  <suite name="Test Suite" duration-ms="5028" started-at="2015-11-25T02:46:41Z" finished-at="2015-11-25T02:46:46Z">
    <groups>
      <group name="Regression">
        <method signature="TrialClass.testExecution(com.pstc.project.models.data.TestMap)[pri:0, instance:com.pstc.project.tests.TrialClass@1babb4d7]" name="testExecution" class="com.pstc.project.tests.TrialClass"/>
      </group> <!-- Regression -->
      <group name="LoginSurveyMonkey">
        <method signature="TrialClass.testExecution(com.pstc.project.models.data.TestMap)[pri:0, instance:com.pstc.project.tests.TrialClass@1babb4d7]" name="testExecution" class="com.pstc.project.tests.TrialClass"/>
      </group> <!-- LoginSurveyMonkey -->
    </groups>
    <test name="Test1" duration-ms="5028" started-at="2015-11-25T02:46:41Z" finished-at="2015-11-25T02:46:46Z">
      <class name="com.pstc.project.tests.TrialClass">
        <test-method status="PASS" signature="TrialClass1(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.pstc.project.tests.TrialClass@1babb4d7]" name="TrialClass1" is-config="true" duration-ms="0" started-at="2015-11-25T08:16:41Z" finished-at="2015-11-25T08:16:41Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[chrome]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[windows]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TrialClass1 -->
        <test-method status="FAIL" signature="testExecution(com.pstc.project.models.data.TestMap)[pri:0, instance:com.pstc.project.tests.TrialClass@1babb4d7]" name="testExecution" duration-ms="4746" started-at="2015-11-25T08:16:41Z" data-provider="SurveyMonkey_DP" finished-at="2015-11-25T08:16:46Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[com.pstc.project.models.data.TestMap@26d777db]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
	at com.pstc.framework.FrameworkServices.getWebDriverInstance(FrameworkServices.java:150)
	at com.pstc.project.tests.TrialClass.testExecution(TrialClass.java:45)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.TestMethodWithDataProviderMethodWorker.call(TestMethodWithDataProviderMethodWorker.java:73)
	at org.testng.internal.TestMethodWithDataProviderMethodWorker.call(TestMethodWithDataProviderMethodWorker.java:14)
	at java.util.concurrent.FutureTask.run(FutureTask.java:262)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:471)
	at java.util.concurrent.FutureTask.run(FutureTask.java:262)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:744)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
            <line>
              <![CDATA[Unable to launch browser instance due to following exception : Could not start a new session. Possible causes are invalid address of the remote server or browser start-up failure.
Build info: version: '2.48.2', revision: '41bccdd10cf2c0560f637404c2d96164b67d9d67', time: '2015-10-09 13:08:06'
System info: host: 'D1ML14605', ip: '192.168.1.3', os.name: 'Windows 8', os.arch: 'amd64', os.version: '6.2', java.version: '1.7.0_45'
Driver info: driver.version: RemoteWebDriver]]>
            </line>
          </reporter-output>
        </test-method> <!-- testExecution -->
        <test-method status="FAIL" signature="testExecution(com.pstc.project.models.data.TestMap)[pri:0, instance:com.pstc.project.tests.TrialClass@1babb4d7]" name="testExecution" duration-ms="4730" started-at="2015-11-25T08:16:41Z" data-provider="SurveyMonkey_DP" finished-at="2015-11-25T08:16:46Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[com.pstc.project.models.data.TestMap@210c2ed8]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
	at com.pstc.framework.FrameworkServices.getWebDriverInstance(FrameworkServices.java:150)
	at com.pstc.project.tests.TrialClass.testExecution(TrialClass.java:45)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.TestMethodWithDataProviderMethodWorker.call(TestMethodWithDataProviderMethodWorker.java:73)
	at org.testng.internal.TestMethodWithDataProviderMethodWorker.call(TestMethodWithDataProviderMethodWorker.java:14)
	at java.util.concurrent.FutureTask.run(FutureTask.java:262)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:471)
	at java.util.concurrent.FutureTask.run(FutureTask.java:262)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:744)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
            <line>
              <![CDATA[Unable to launch browser instance due to following exception : Could not start a new session. Possible causes are invalid address of the remote server or browser start-up failure.
Build info: version: '2.48.2', revision: '41bccdd10cf2c0560f637404c2d96164b67d9d67', time: '2015-10-09 13:08:06'
System info: host: 'D1ML14605', ip: '192.168.1.3', os.name: 'Windows 8', os.arch: 'amd64', os.version: '6.2', java.version: '1.7.0_45'
Driver info: driver.version: RemoteWebDriver]]>
            </line>
          </reporter-output>
        </test-method> <!-- testExecution -->
        <test-method status="FAIL" signature="testExecution(com.pstc.project.models.data.TestMap)[pri:0, instance:com.pstc.project.tests.TrialClass@1babb4d7]" name="testExecution" duration-ms="4761" started-at="2015-11-25T08:16:41Z" data-provider="SurveyMonkey_DP" finished-at="2015-11-25T08:16:46Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[com.pstc.project.models.data.TestMap@bfddec9]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
	at com.pstc.framework.FrameworkServices.getWebDriverInstance(FrameworkServices.java:150)
	at com.pstc.project.tests.TrialClass.testExecution(TrialClass.java:45)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.TestMethodWithDataProviderMethodWorker.call(TestMethodWithDataProviderMethodWorker.java:73)
	at org.testng.internal.TestMethodWithDataProviderMethodWorker.call(TestMethodWithDataProviderMethodWorker.java:14)
	at java.util.concurrent.FutureTask.run(FutureTask.java:262)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:471)
	at java.util.concurrent.FutureTask.run(FutureTask.java:262)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:744)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
            <line>
              <![CDATA[Unable to launch browser instance due to following exception : Could not start a new session. Possible causes are invalid address of the remote server or browser start-up failure.
Build info: version: '2.48.2', revision: '41bccdd10cf2c0560f637404c2d96164b67d9d67', time: '2015-10-09 13:08:06'
System info: host: 'D1ML14605', ip: '192.168.1.3', os.name: 'Windows 8', os.arch: 'amd64', os.version: '6.2', java.version: '1.7.0_45'
Driver info: driver.version: RemoteWebDriver]]>
            </line>
          </reporter-output>
        </test-method> <!-- testExecution -->
        <test-method status="FAIL" signature="testExecution(com.pstc.project.models.data.TestMap)[pri:0, instance:com.pstc.project.tests.TrialClass@1babb4d7]" name="testExecution" duration-ms="4761" started-at="2015-11-25T08:16:41Z" data-provider="SurveyMonkey_DP" finished-at="2015-11-25T08:16:46Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[com.pstc.project.models.data.TestMap@63781bf1]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
	at com.pstc.framework.FrameworkServices.getWebDriverInstance(FrameworkServices.java:150)
	at com.pstc.project.tests.TrialClass.testExecution(TrialClass.java:45)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.TestMethodWithDataProviderMethodWorker.call(TestMethodWithDataProviderMethodWorker.java:73)
	at org.testng.internal.TestMethodWithDataProviderMethodWorker.call(TestMethodWithDataProviderMethodWorker.java:14)
	at java.util.concurrent.FutureTask.run(FutureTask.java:262)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:471)
	at java.util.concurrent.FutureTask.run(FutureTask.java:262)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:744)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
            <line>
              <![CDATA[Unable to launch browser instance due to following exception : Could not start a new session. Possible causes are invalid address of the remote server or browser start-up failure.
Build info: version: '2.48.2', revision: '41bccdd10cf2c0560f637404c2d96164b67d9d67', time: '2015-10-09 13:08:06'
System info: host: 'D1ML14605', ip: '192.168.1.3', os.name: 'Windows 8', os.arch: 'amd64', os.version: '6.2', java.version: '1.7.0_45'
Driver info: driver.version: RemoteWebDriver]]>
            </line>
          </reporter-output>
        </test-method> <!-- testExecution -->
      </class> <!-- com.pstc.project.tests.TrialClass -->
      <class name="com.pstc.framework.FrameworkSetupSuiteLevel">
        <test-method status="PASS" signature="BeforeSuite()[pri:0, instance:com.pstc.framework.FrameworkSetupSuiteLevel@4f62cf6a]" name="BeforeSuite" is-config="true" duration-ms="1915" started-at="2015-11-25T08:16:39Z" finished-at="2015-11-25T08:16:41Z">
          <reporter-output>
            <line>
              <![CDATA[App Link:-https://www.surveymonkey.com/]]>
            </line>
            <line>
              <![CDATA[Before Suite Execution Finish]]>
            </line>
          </reporter-output>
        </test-method> <!-- BeforeSuite -->
        <test-method status="PASS" signature="AfterSuite()[pri:0, instance:com.pstc.framework.FrameworkSetupSuiteLevel@4f62cf6a]" name="AfterSuite" is-config="true" duration-ms="0" started-at="2015-11-25T08:16:46Z" finished-at="2015-11-25T08:16:46Z">
          <reporter-output>
            <line>
              <![CDATA[After Suite Execution Finish]]>
            </line>
          </reporter-output>
        </test-method> <!-- AfterSuite -->
      </class> <!-- com.pstc.framework.FrameworkSetupSuiteLevel -->
    </test> <!-- Test1 -->
  </suite> <!-- Test Suite -->
</testng-results>
